packages/embark-specialconfigs/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "embark-specialconfigs",
```

Comment:
Probably too late now, but I'd really like us to see using `@embark/package-name` instead

Reply:
I don't think `@embark` will ever be available to us. But, yesterday an `embark-framework` org was created on npmjs.com and org's automatically reserve a matching namespace. So we could to `@embark-framework/specialconfigs`.

Reply:
Agreed yes. Not really hoping for `@embark` specifically anymore, but some scope would be great :)